Dockerfile

```diff
@@ -48,31 +48,27 @@ ENV NEXUS_HOME=${SONATYPE_DIR}/nexus \
     SONATYPE_WORK=${SONATYPE_DIR}/sonatype-work \
     DOCKER_TYPE='rh-docker'
 
-ARG NEXUS_REPOSITORY_MANAGER_COOKBOOK_VERSION="release-0.5.20220111-153152.2b86c3a"
-ARG NEXUS_REPOSITORY_MANAGER_COOKBOOK_URL="https://github.com/sonatype/chef-nexus-repository-manager/releases/download/${NEXUS_REPOSITORY_MANAGER_COOKBOOK_VERSION}/chef-nexus-repository-manager.tar.gz"
+# Install Java & tar
+RUN microdnf update -y \
+    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-1.8.0-openjdk-headless tar \
+    && microdnf clean all \
+    && groupadd --gid 200 -r nexus \
+    && useradd --uid 200 -r nexus -g nexus
```

Comment:
`200` is specifically referenced in the description, the IQ container has moved away from this we may want to consider doing this in the future

Reply:
Will this change the uid/gid? For customers that have been running previous versions of the container, the file ownership on their mounted volumes depend the uid/gid staying constant. Lifecycle changed uid/gid a while back and it resulted in the need for a manual upgrade step (see https://help.sonatype.com/iqserver/product-information/release-notes/2020-release-notes#id-2020ReleaseNotes-DockerImageUserPermissionsMigration).

Reply:
The Hub documentation says `200` https://hub.docker.com/r/sonatype/nexus3

Although upon closer inspection it does say user, so the GID may be arbitrary

Reply:
https://github.com/sonatype/chef-nexus-repository-manager/blob/e8c04ee607de36a583348a0d2faa403cf880b702/recipes/users.rb#L8

Reply:
Ah nevermind - I saw what you mean in the README, 200 is the uid/gid already in use, so no change. 👍